includes/class-wc-stripe-webhook-handler.php

```diff
@@ -768,6 +770,25 @@ public function is_partial_capture( $notification ) {
 		return 0 < $notification->data->object->amount_refunded;
 	}
 
+	/**
+	 * Gets the first refund object from charge notification.
+	 *
+	 * @since x.x.x
+	 * @param object $notification
+	 *
+	 * @return object
+	 */
+	public function get_refund_object( $notification ) {
+		// Since API version 2022-11-15, the Charge object no longer expands `refunds` by default.
+		// We can remove this once we drop support for API versions prior to 2022-11-15.
+		if ( ! empty( $notification->data->object->refunds->data[0] ) ) {
+			return $notification->data->object->refunds->data[0];
+		}
```

Comment:
This isn't required as the snippet below should expand the refunds list from the Charge object, but it's one less request and we can keep it until we drop support for API versions prior to 2022-11-15.